### DISCUSSION TITLE
App not loading after upgrade to solara==1.25.0

### DISCUSSION THREAD

**Discussion by francesfeng** [widgetti/solara#430] [Q&A] [answered]
Hi everyone, 

After I upgraded solara from 1.22.0 to 1.25.0, the app doesn't load at all. After some digging, seems a possible reason is that React using development built? The problem persists after I downgrade to 1.24.0 and 1.23.0, but if I use pip install "solara==1.22.0", the app loads again. Anyone has similar issue?

**Comment by maartenbreddels** [comment, marked as answer]
Hi,

could you open the dev console (F12 in many browser), make sure the clear cache is on in the network tab, and refresh?.
I think it's time we fix this!

cheers,

Maarten

**Reply by francesfeng** [reply]
Thanks Maarten for the prompt reply. 

Yes, the problem resolved now. I cleared all browsing data from chrome, and the app loaded!

**Comment by maartenbreddels** [comment]
Newer versions shouldn't have this problem anymore, if this is still required, let us know.
